students/mielczarek_szymon/lesson_06_dicts_tuples_sets_args_kwargs/frequency_analysis.py

```diff
@@ -0,0 +1,13 @@
+from operator import itemgetter
+
+n = int(input())
+words_freq = {}
+for _ in range(n):
+    for word in input().split():
+        words_freq[word] = words_freq.get(word, 0) + 1
+
+words_list = [(val, key) for key, val in words_freq.items()]
+words_list = sorted(sorted(words_list, key=itemgetter(1)), key=itemgetter(0),
```

Comment:
in this line i would use different name to reflect its contents, like `sorted_words_list`

Reply:
you can also try out tuple sorting properties:
`sorted_list = sorted([(-count, word) for word, count in words_list.items()])`